src/main/java/com/github/princesslana/eriscasper/data/resource/AuditLogChange.java

```diff
@@ -0,0 +1,17 @@
+package com.github.princesslana.eriscasper.data.resource;
+
+import java.util.Optional;
+import org.immutables.value.Value;
+
+@SuppressWarnings("unused")
+@Value.Immutable
+public interface AuditLogChange {
+
+  boolean isArray();
+
+  Optional<Object> newValue();
```

Comment:
`getNewValue` and `getOldValue` would be more consistent with other classes.

Reply:
Agreed, just changed fields -> methods my mistake, also need to make tests for it.